sdk/ai/azure-ai-projects/samples/evaluation/sample_evaluations_async.py

```diff
@@ -60,20 +58,37 @@ async def main() -> None:
 
             print("Create an evaluation")
             evaluation: Evaluation = Evaluation(
-                display_name="Sample Evaluation",
-                description="Sample evaluation for testing",  # TODO: Can we optional once bug 4115256 is fixed
-                data=InputDataset(id="<dataset_id>"),  # TODO: update this to use the correct id
+                display_name="Sample Evaluation Async",
+                description="Sample evaluation for testing",
+                # Sample Dataset Id : azureai://accounts/<account_name>/projects/<project_name>/data/<dataset_name>/versions/<version>
+                data=InputDataset(id="<>"),
                 evaluators={
                     "relevance": EvaluatorConfiguration(
-                        id=EvaluatorIds.RELEVANCE.value,  # TODO: update this to use the correct id
+                        id=EvaluatorIds.RELEVANCE.value,
                         init_params={
-                            "deployment_name": "gpt-4o",
+                            "deployment_name": model_deployment_name,
                         },
+                        data_mapping={
+                            "query": "${data.query}",
+                            "response": "${data.response}",
+                        },
+                    ),
+                    "violence": EvaluatorConfiguration(
+                        id=EvaluatorIds.VIOLENCE.value,
+                        init_params={
+                            "azure_ai_project": endpoint,
+                        },
+                    ),
+                    "bleu_score": EvaluatorConfiguration(
+                        id=EvaluatorIds.BLEU_SCORE.value,
                     ),
                 },
             )
 
-            evaluation_response: Evaluation = await project_client.evaluations.create(evaluation)
+            evaluation_response: Evaluation = await project_client.evaluations.create(evaluation, headers={
```

Comment:
will this pass empty header if there is no deployment?

Reply:
These are coming from env vars at top of the sample. So assumption is those are available and set before running the sample.